code/script.js

```diff
@@ -69,36 +69,44 @@ const WhatToEat = (name) => {
 let Pasta = document.getElementById('Pasta');
 Pasta.addEventListener('click', (event) => {
   event.preventDefault()
-  showMessage("Fab choice! Here is a recipe for a creamy Pasta Carbonara", 'bot')
+  showMessage("Pasta sounds good!", 'user')
+  showMessage("Pasta it is! I think you would enjoy a creamy Pasta Carbonara", 'bot')
   setTimeout(() => FoodChoice(event.target.value), 1200)
+  inputWrapper.style.display = 'none';
 });
 
 let Salad = document.getElementById('Salad');
 Salad.addEventListener('click', (event) => {
   event.preventDefault()
-  showMessage("Fab choice! Here is a recipe for a delish Ceasar salad", 'bot')
+  showMessage("Salad sounds good!", 'user')
+  showMessage("So, a salad. You would probably enjoy a creamy Ceasar Salad.", 'bot')
```

Comment:
I really like that the user answer shows up instantly. Maybe the bot answers can be delayed a bit with the setTimeout function?